### ISSUE TITLE
Support for pretrainedVectors

### ISSUE THREAD

**Issue by naushadzaman** [mklf/FastTextPy#1]
Support for pretrainedVectors is missing. Is there any easy way of adding this support?

**Comment by mklf** [comment]
you can use `sg = FastText().load("filename")` to load pretrained model of binary format

**Comment by naushadzaman** [comment]
I mean adding pretrainedVector in training, e.g. 

sup = FastText().train(train_file,dim=10,model='supervised',lr=0.1,word_ngrams=2,minn=3, maxn=6, min_count=1,bucket=10000000,epoch=5,thread=4, pretrainedVector=filepath_to_pretrained_vector)

**Comment by naushadzaman** [comment]
https://github.com/facebookresearch/fastText

  The following arguments for training are optional:
  -lr                 learning rate [0.05]
  -lrUpdateRate       change the rate of updates for the learning rate [100]
  -dim                size of word vectors [100]
  -ws                 size of the context window [5]
  -epoch              number of epochs [5]
  -neg                number of negatives sampled [5]
  -loss               loss function {ns, hs, softmax} [ns]
  -thread             number of threads [12]
  -pretrainedVectors  pretrained word vectors for supervised learning []
  -saveOutput         whether output params should be saved [0]
